doc/build_apps/js_app_bundle.rst

```diff
@@ -152,3 +152,15 @@ An app bundle can be wrapped into a governance proposal with the Python client f
 
 Once :ref:`submitted and accepted <governance/proposals:Submitting a New Proposal>`, a ``set_js_app`` proposal atomically (re-)deploys the complete JavaScript application.
 Any existing application endpoints and JavaScript modules are removed.
+
+By default, the source code is pre-compiled into bytecode and both the source code and the bytecode are stored in the Key Value store. To disable precompilation and remove any existing cached bytecode, add ``--disable-bytecode-cache`` to the above command. See :ref:`Resource Usage <operations/resource_usage:Memory>` for a discussion on latency vs. memory usage.
```

Comment:
We should list the new tables in https://microsoft.github.io/CCF/main/audit/builtin_maps.html, which should have definitely been picked up by the `governance_history.test_tables_doc` step in the `test_suite.py` end-to-end test, but didn't because we run the test against the C++ app.

We should really run the test suite against the logging JS app instead. Could you run the full test suite with the JS app (changing this line https://github.com/microsoft/CCF/blob/main/tests/e2e_suite.py#L196) and create a ticket for what breaks?

Reply:
Well spotted!

Reply:
I've added docs for those tables. And I've filed https://github.com/microsoft/CCF/issues/2689.